.gitignore

```diff
@@ -0,0 +1,12 @@
+# IntelliJ files
```

Comment:
Could we add the name of the `.jar` file to the `.gitignore` file so it doesn't appear in the staged area when doing `git status`

Reply:
addressed in df34d9a